python/py-hdbscan/Portfile

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        scikit-learn-contrib hdbscan 0.8.24
+name                py-${github.project}
+revision            0
+categories-append   science
+platforms           darwin
+license             BSD
+
+python.versions     35 36 37 38
```

Comment:
Let's only add the current pythons for new python ports. So please remove the py35 and py36 sub-ports.

Reply:
Thanks done.

Reply:
it seems that the py35/py36 subports have reappeared now?

Reply:
Agreed. Please remove them again.

Reply:
Whoops sorry. Done.

Reply:
Aside: What's the git command to compare to the remote? It said I was up-to-date, but edits across multiple boxes fouled things up, and without seeing a warning.